app/views/courses/course/_section-lessons.html.erb

```diff
@@ -6,9 +6,9 @@
         <%= numbered_lesson_title(lesson, lesson_index) %>
       <% end %>
 
-      <% if user_signed_in? %>
-          <i class="fa fa-check-circle-o section-lessons__item__icon <%= lesson_completed?(@user, lesson) %>"></i>
-      <% end %>
+      <div id="section-lessons__<%= lesson.id %>">
```

Comment:
can we wrap this div in a check to make sure the user exists before rendering it please?
 `<% if user_signed_in? %>`

I know you have that in the partial already, the two levels of if statements in that file make it a bit difficult to read though.